jupyter_elfinder/elfinder.py

```diff
@@ -1072,7 +1113,8 @@ def __info(self, path: str) -> Info:
                         tmb_url = self.__path2url(tmb)
                         info["tmb"] = tmb_url
                     else:
-                        self._response["tmb"] = True
+                        # FIXME: This is not part of api 2.1
+                        self._response[R_TMB] = True
```

Comment:
This also doesn't look to be part of api 2.1. See https://github.com/Studio-42/elFinder/wiki/Client-Server-API-2.1#info

Reply:
This is the spec say: if file do not have thumbnail yet, but it can be generated than it must have value "1"

So we should change True to 1